Story/vspc 42

renilJoseph opened the pull request:
please verify the code.

diging-jenkins commented:
Can one of the admins verify this patch?

jdamerow commented:
Please see my previous comments. Also, please reopen previous pull requests to keep the comment history.

renilJoseph commented:
Previous branch was deleted, so cannot reopen it.
As per comment, added font awesome globe icon, changed c:url to href, set external link to open in new window and set the color to red. Please verify.

jdamerow commented:
Jenkins, test this please.

jdamerow commented on vspace/src/main/webapp/WEB-INF/views/exhibition/space.jsp:
```diff
@@ -42,6 +42,29 @@ function drawLinks() {
+        link.attr('href', "${link.externalLink.externalLink}");
+        link.attr('target', "_blank");
+        
+        var linkDisplay = $('<span data-feather="navigation-2" class="fa fa-globe"></span>');
```
did you notice that the icon is not shown? you need to remove the feather icon, or the globe icon won't show up.

renilJoseph replied:
yes. I missed that. Thank you.

jdamerow commented:
Jenkins, test this please.
